src/pages/debrief/debrief.ts

```diff
@@ -237,7 +237,10 @@ export class DebriefPage extends PracticeableBasePageComponent {
       this.navController.push('PassFinalisationPage');
       return;
     }
-    this.navController.push('BackToOfficePage');
+    this.navController.push('BackToOfficePage').then(() => {
+      // remove Test Report Page and Debrief Page
+      this.navController.remove(this.navController.getActive().index - 2, 2);
```

Comment:
Are we able to remove pages by name rather than an index? This index to page relation will vary between categories.

Reply:
Tom asked the same thing. I can probably use `this.navController.indexOf('TestReportPage')` to get the index and call remove multiple times.

Reply:
Let me just see if that works.

Reply:
@ammarhaiderbjss No, not that easy unfortunately. Let me see what other ways there are